core/src/test/java/io/confluent/kafka/schemaregistry/ClusterTestHarness.java

```diff
@@ -107,6 +107,7 @@ public static int choosePort() {
   protected List<KafkaServer> servers = null;
   protected String brokerList = null;
 
+  protected int schemaRegistryPort ;
```

Comment:
A couple of minor formatting issues in this file's diff -- extra space at the end of this line, and on `getSchemaRegistryProtocol` there's a missing space before the `{`.